### ISSUE TITLE
IconImage dispose itself without dispose the underlying SKBitmap object

### ISSUE THREAD

**Issue by zhuxb711** [ComputerGhost/FaviconFetcher#30]
Just set _bitmap into null

https://github.com/ComputerGhost/FaviconFetcher/blob/4d02448fe04ef64a7a72f4a00340bda049ed0b96/FaviconFetcher/Classes/IconImage.cs#L72

If concerning dispose the _bitmap will affect `ToSKBitmap()`. 
I suggest to change `ToSKBitmap()` to:

```csharp
public SKBitmap ToSKBitmap()
{
    return _bitmap.Copy();
}
```

or 

```csharp
// Developer should be responsible for disposing an 'Underlying' object which might cause some problems
public SKBitmap GetUnderlyingSKBitmap()
{
    return _bitmap;
}
```

**Comment by kiddailey** [comment]
Nice catch, thank you!  Would it be simpler to just properly dispose the SKBitmap properly?

Returning a copy of the bitmap could increase memory usage if I'm understanding correctly.

**Comment by zhuxb711** [comment]
Yes sure, just dispose the underlying _bitmap is ok for this case. SKBitmap should be disposed when IconImage disposing

**Comment by zhuxb711** [comment]
Will this be fixed in the next release?
